source/postprocess/visualization/adiabat.cc

```diff
@@ -33,7 +33,8 @@ namespace aspect
       Adiabat<dim>::
       Adiabat ()
         :
-        DataPostprocessor<dim> ()
+        DataPostprocessor<dim> (),
+        Interface<dim>("units")
```

Comment:
This postprocessor produces output for a variety of quantities that don't have common units. Let's just pass the empty string in that case.
```suggestion
        Interface<dim>("")
```